torchvision/prototype/transforms/_geometry.py

```diff
@@ -0,0 +1,75 @@
+from typing import Any, Dict, List, Union, Sequence, Tuple
+
+from torchvision import transforms as _transforms
+from torchvision.prototype.transforms import Transform, InterpolationMode
+
+from . import functional as F
+from .utils import Query
+
+
+class HorizontalFlip(Transform):
+    _DISPATCHER = F.horizontal_flip
+
+
+class Resize(Transform):
+    _DISPATCHER = F.resize
+
+    def __init__(
+        self,
+        size: Union[int, Sequence[int]],
+        interpolation: InterpolationMode = InterpolationMode.BILINEAR,
+    ) -> None:
+        super().__init__()
+        self.size = size
+        self.interpolation = interpolation
+
+    def get_params(self, sample: Any) -> Dict[str, Any]:
+        return dict(size=self.size, interpolation=self.interpolation)
```

Comment:
> On the previous API, the `get_params` method used to return only the randomly generated parameters. Here it seems that we use it for all of them. Could you confirm that this is the intention? (we should document this prior merging this)

That is intentional, yes. The issue is that due to the automatic dispatch, we have no other "hook" to define which parameters get passed to the dispatchers.

> Also do you think it's good to keep the same name for the function if we change its behaviour? For example to avoid confusion we could name it `_get_args` or something similar. The motivation here is to not confuse this new method with the old static one.

Good point. If we make it private as suggested in https://github.com/pytorch/vision/pull/5418#discussion_r805937596, we should be good.